pandas/tests/test_base.py

```diff
@@ -998,6 +998,42 @@ def test_validate_bool_args(self):
             with pytest.raises(ValueError):
                 self.int_series.drop_duplicates(inplace=value)
 
+    def test_get(self):
+        for o in self.objs:
+            if isinstance(o, Series):
+                s = o.set_axis([2 * i for i in range(len(o))], inplace=False)
```

Comment:
just directly construct a Series with the provided values rather than trying to convolute this

Reply:
done